README.md

```diff
@@ -314,6 +314,40 @@ $rt->report([
 ]);
 ```
 
+## Options
+
+When using the MinFraud client in PHP, you have the flexibility to customize its behavior by providing various options. The options parameter is used to pass these customizations to the client when instantiating it.
+
+| Option           | Description                                                                                                                                                                          |
+|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **host**         | The host to use when connecting to the web service. By default, the client connects to the production host. However, during testing and development, you can set this option to '[sandbox.maxmind.com](sandbox.maxmind.com)' to use the sandbox host. The sandbox host allows you to experiment with the API without affecting your production data. |
```

Comment:
I think this kind of stuff would belong in the API docs. See [here](https://maxmind.github.io/minfraud-api-php/doc/v1.23.0/classes/GeoIp2-WebService-Client.html#method___construct) for how we mention the alternate host for the GeoIP2-php API.